geometry_msgs/Twist twist
bool[6] mask
